packages/core/src/internal/utils/validators.ts

```diff
@@ -145,6 +145,14 @@ export function validateDisposableTokenExpiry(expiresIn: ExpiresIn) {
   }
 }
 
+export function validateDisposableTokenTokenID(tokenID: string) {
+  if (tokenID.length > 64) {
+    throw new InvalidArgumentError(
+      'TokenID must be less than or equal to 64 characters.'
+    );
+  }
```

Comment:
I had thought to `encode` the string and check the byte length in UTF-8 or something so lmk if this seems less potent that that.

Reply:
I think this is fine